<launch>
    <node pkg="rolling_window" type="rolling_window_pcl" name="rolling_window_pcl" output="screen">
	<remap from="scan_in" to="sick_scan2" />
	<param name="odom_trigger_thresh" value="0.5" type="double" />
	<param name="front_bound" value="10.0" type="double" />
	<param name="back_bound" value="-3.0" type="double" />
    </node>

    <node pkg="rolling_window" type="road_surface" name="road_surface" output="screen">
	<!--param name="search_radius" value="0.3" type="double" /-->
	<param name="search_radius" value="0.3" type="double" />
	<param name="curvature_thresh" value="0.005" type="double" />
	<param name="curvature_visual_limit" value="0.1" type="double" />
 	<param name="normalZ_visual_limit_" value="0.2" type="double" />
	<param name="curvature_visualization" value="true"/>
	<param name="normalZ_visualization" value="false"/>
    </node>
</launch>    
